docs/sources/plugins/catalog.md

```diff
@@ -6,63 +6,66 @@ weight = 1
 
 # Plugin catalog
 
-The plugin catalog allows you to browse and manage plugins from within Grafana. Only a Grafana server administrator can access and use the catalog.
+The plugin catalog allows you to browse and manage plugins from within Grafana. Only Grafana server administrators and organization administrators can access and use the catalog. The following access rules apply depending on the user role
+
+| Org Admin | Server Admin | Permissions                                                                                 |
+| --------- | ------------ | ------------------------------------------------------------------------------------------- |
+| [x]       | [x]          | <ul><li>Can configure app plugins</li><li>Can install/uninstall/update plugins</li></ul>    |
+| [x]       | [ ]          | <ul><li>Can configure app plugins</li><li>Cannot install/uninstall/update plugins</li></ul> |
+| [ ]       | [x]          | <ul><li>Cannot configure app plugins</li><li>Can install/uninstall/update plugins</li></ul> |
 
 > **Note:** The plugin catalog is designed to work with a single Grafana server instance only. Support for Grafana clusters will be added in future Grafana releases.
 
 <div class="medium-6 columns">
   <video width="700" height="600" controls>
-    <source src="/static/assets/videos/plugins-catalog-install-8-0.mp4" type="video/mp4">
+    <source src="/static/assets/videos/plugins-catalog-install-8-1.mp4" type="video/mp4">
     Your browser does not support the video tag.
   </video>
 </div>
 
 Before you can use the plugin catalog, you must enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#plugin_admin_enabled" >}}) file.
 Before following the steps below, make sure you are logged in as a Grafana administrator.
 
+<a id="#plugin-catalog-entry"></a>
+There are currently two entry points to the Plugin catalog. Grafana server administrators can find the Plugin catalog at **Server Admin > Plugins**. Organization administrators can find the Plugin catalog at **Configuration > Plugins**.
```

Comment:
Is this "Plugin catalog" or "plugin catalog"? Either way, let's try to be consistent in docs.